server/database/queries/updateUserQuery.js

```diff
@@ -0,0 +1,13 @@
+const connection = require('../config/connection');
+
+const updateUserQuery = ({
+  firstName, lastName, mobile, address, userId,
+}) => {
+  const sql = {
+    text: 'UPDATE users SET first_name = $1,last_name = $2, mobile =$3, address = $4 WHERE id = $5  RETURNING *;',
```

Comment:
- if you can add more field to update will be excellent like `country, city, and  alt_mobile`